.gitlab-ci/lint.yml

```diff
@@ -6,6 +6,17 @@ yamllint:
     - yamllint --strict .
   except: ['triggers', 'master']
 
+vagrant-validate:
+  extends: .job
+  stage: unit-tests
+  script:
+    - apt-get update
+    - apt-get install virtualbox -y
+    - curl -sL https://releases.hashicorp.com/vagrant/2.2.4/vagrant_2.2.4_x86_64.deb -o /tmp/vagrant_2.2.4_x86_64.deb
+    - dpkg -i /tmp/vagrant_2.2.4_x86_64.deb
+    - vagrant validate
```

Comment:
You can do `vagrant validate --ignore-provider` and skip the virtualbox install.

Reply:
Ofc...... that worked!